daemon/mgr/container.go

```diff
@@ -1302,6 +1305,33 @@ func (mgr *ContainerManager) Resize(ctx context.Context, name string, opts types
 	return mgr.Client.ResizeContainer(ctx, c.ID, opts)
 }
 
+// Wait stops processing until the given container is stopped.
+func (mgr *ContainerManager) Wait(ctx context.Context, name string) (types.ContainerWaitOKBody, error) {
+	c, err := mgr.container(name)
+	if err != nil {
+		return types.ContainerWaitOKBody{}, err
+	}
+
+	// We should notice that container's meta data shouldn't be locked in wait process, otherwise waiting for
+	// a running container to stop would make other client commands which manage this container are blocked.
+	// If a container status is exited or stopped, return exit code immediately.
+	if c.IsExited() || c.IsStopped() {
+		return types.ContainerWaitOKBody{
+			Error:      c.State.Error,
+			StatusCode: c.ExitCode(),
+		}, nil
+	}
+	// If a container status is created, return 0 as status code.
+	if c.IsCreated() {
+		return types.ContainerWaitOKBody{
+			Error:      c.State.Error,
```

Comment:
I think we can just left the Error field empty. WDYT?

Reply:
OK, I'll fix all.